floss/main.py

```diff
@@ -349,19 +357,24 @@ def select_functions(vw, asked_functions: Optional[List[int]]) -> Set[int]:
     return asked_functions_
 
 
-def is_supported_file_type(sample_file_path: Path):
+def get_file_type(sample_file_path: Path):
     """
-    Return if FLOSS supports the input file type, based on header bytes
+    Returns input file type, based on header bytes
     :param sample_file_path:
-    :return: True if file type is supported, False otherwise
+    :return: file type if it is supported, raise Error Otherwise
     """
     with sample_file_path.open("rb") as f:
-        magic = f.read(2)
+        magic = f.read(4)
 
     if magic in SUPPORTED_FILE_MAGIC:
-        return True
+        return magic
```

Comment:
could then use an Enum here to return ELF or PE (or constants)

Reply:
We want to return ELF/PE magic string as the name of the function is get_file_type. There are some places where we need PE and ELF specific execution, at those places I am checking file type with the help of this function and SUPPORTED_FILE_MAGIC_PE/ELF constants. 

Do you want me to convert SUPPORTED_FILE_MAGIC_PE /ELF to enum?